_posts/2021-05-26-cockpit-245.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Cockpit 245
+author: lis
+date: 2021-05-26
+tags: cockpit linux
+slug: cockpit-245
+category: release
+summary: Cockpit with metrics and storage improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 245 and cockpit-podman version 31.
+
+## Release process: No more cockpit-cache tarball
+
+The Cockpit project is no longer distributing the cockpit-cache tarball as part
+of its releases.  This tarball previously contained the exact state of the
+`node_modules/` directory at the time the release was made, and was provided for
+reproducibility and possible GPL compliance reasons (since the main release
+tarball contains a good deal of generated code).
+
+This has been replaced with a separate git repository,
+[cockpit-project/node-cache](https://github.com/cockpit-project/node-cache/releases),
+which will now receive a tag for each release.
+
+## Metrics: New PCP configuration dialog
+
+The Metrics page header has a new "Metrics settings" button:
+
+![Metrics page header showing new settings button](/images/metrics-menu.png)
+
+This allows you to enable or disable the [Performance Co-Pilot](https://pcp.io/) (PCP) metrics collector:
+
+![The start of a new Metrics settings dialog](/images/metrics-settings.png)
+
+Related options will be added in the near future.
+
+## Storage: Show both SHA256 and SHA1 Tang fingerprints
+
+Newer versions of `tang-show-keys` will use SHA256 to compute the thumbprints
+of the keys, so Cockpit now shows both versions to allow key verification for
+both new and old versions.
+
+![Key verification with SHA256 and SHA1 fingerprints](/images/storage-tang-dual-fingerprint.png)
+
+## cockpit-podman: Added Korean translation
+
+A new translation has been added for the Korean language.
+
+Thank you to [simmon](https://translate.fedoraproject.org/user/simmon/) for the efforts.
```

Comment:
```suggestion
Thanks to [simmon](https://translate.fedoraproject.org/user/simmon/) for the efforts.
```

Reply:
GH account: https://github.com/simmon-nplob
But weblate account probably makes equal sense here

Reply:
I had the GitHub account on the first draft, in fact, but I switched it over to Weblate.